src/smtsolvers/CoreSMTSolver.cc

```diff
@@ -231,11 +206,10 @@ Var CoreSMTSolver::newVar(bool sign, bool dvar)
     vardata  .push(mkVarData(CRef_Undef, 0));
     activity .push(rnd_init_act ? drand(random_seed) * 0.00001 : 0);
     seen     .push(0);
-    polarity .push(sign);
     decision .push();
     trail    .capacity(v+1);
     setDecisionVar(v, dvar);
-    polarity    .push((char)sign);
+    savedPolarity.push(true);
 
 #if CACHE_POLARITY
     prev_polarity.push(toInt(l_Undef));
```

Comment:
I saw in `CoreSMTSolver.h` that member `prev_polarity` (together with the macro) was deleted, so I believe this should also be removed from here, no?